src/jetson/main.py

```diff
@@ -121,6 +124,7 @@ def detect(self,
                 bboxes.append((xmin, ymin, xmax, ymax))
             return bboxes
 
+
 class VideoCapturer(object):
```

Comment:
please include a docstring here

Reply:
docstring containing VideoCapturer class information is located in __init__ function. Should this be moved? This is the case for all classes in this script.